lib/src/Base/Func/NormInfEnumerateFunction.cxx

```diff
@@ -102,12 +103,32 @@ Indices NormInfEnumerateFunction::operator() (const UnsignedInteger index) const
         ++ index_;
         break;
       }
-    }
-  }
+    } // j
+  } // while (index_ < index)
   return multiIndices_;
 }
 
 
+/* The inverse of the association */
+UnsignedInteger NormInfEnumerateFunction::inverse(const Indices & indices) const
+{
```

Comment:
this needs a dedicated test in t_NormInfEnumerateFunction_std.py

Reply:
ok done